modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -72,6 +82,26 @@ private static class CommittedConfiguration {
         }
     }
 
+    private final FinishTxCommandHandler finishTxCommandHandler;
+
+    /** Constructor. */
+    public ZonePartitionRaftListener(
+            TxStatePartitionStorage txStatePartitionStorage,
+            TxManager txManager,
+            SafeTimeValuesTracker safeTimeTracker,
+            PendingComparableValuesTracker<Long, Void> storageIndexTracker,
+            ZonePartitionId partitionId
```

Comment:
Please rename it to zonePartitionId or replicationGroupId. PartitionId is commonly used (e.g. in storages) as the name for partition index.